index.html

```diff
@@ -0,0 +1,180 @@
+                                <p>Participated in the development of an easy-to-use NFT marketplace and community platform with a focus on art, music, and culture.</p>
+                            </div>
+                            <div class="project-info from-left">
+                                <h3>UCB Quests</h3>
+                                <p>Developed a gamified platform for ‘Unblocked’ featuring quests and tasks, rewarding users with points usable on the Unblocked website.</p>
+                            </div>
+                            <div class="project-info from-right">
+                                <h3>Gun for Glory</h3>
+                                <p>Involved in the front-end development of interactive landing pages using HTML, CSS, and Tailwind CSS.</p>
+                            </div>
+                            <div class="project-info from-right">
+                                <h3>Level Up</h3>
+                                <p>Assisted in creating an engaging landing page with HTML, CSS, and JavaScript.</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="sec5" id="contests">
+                <div class="contests-container">
+                    <div class="contests-title">Contests</div>
+                    <div class="contests">
+                        <div class="contest from-left">
+                            <img src="/Assets/images/pcpc2023.jpeg" alt="PCPC23" class="contest-img">
+                            <div class="contest-info">
+                                <h3>Palestinian Collegiate Programming Contest (PCPC)</h3>
+                                <p>Participated in PCPC22 and PCPC23, securing 7th place in PCPC23, which qualified us for the Arab Collegiate Programming Contest (ACPC) in Egypt. This achievement enhanced our problem-solving and teamwork skills through high-level national and regional competitions.</p>
+                            </div>
+                        </div>
+                        <div class="contest from-right">
+                            <img src="/Assets/images/bits&bytes.jpeg" alt="Bits&Byte" class="contest-img">
+                            <div class="contest-info">
+                                <h3>Bits&Byte Competition</h3>
+                                <p>Won 1st place in the Bits&Byte coding competition, excelling in algorithm optimization and problem-solving.</p>
+                            </div>
+                            <img src="/Assets/icons/badge-icon.svg" class="contest-icon">
+                        </div>
+                        <div class="contest from-left">
+                            <img src="/Assets/images/ieeeXtreem.jpeg" alt="IEEEXtreme" class="contest-img">
+                            <div class="contest-info">
+                                <h3>IEEEXtreme 16.0 & 17.0</h3>
```

Comment:
Please add  `<abbr title="Institute of Electrical and Electronics Engineers Xtreme">IEEEXtreme</abbr>` for acceptability.

Reply:
fixed